Make.inc

```diff
@@ -132,6 +132,10 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Increase these every release when breaking ABI in a backward-incompatible way
```

Comment:
To be more preciseI would say `Increase SOMINOR with every minor release, and SOMAJOR with every major (ABI-breaking) release (and reset SOMINOR to 0)`